src/zh/zaimanhua/src/eu/kanade/tachiyomi/extension/zh/zaimanhua/ZaimanhuaDto.kt

```diff
@@ -170,3 +179,46 @@ data class ImageRetryParamsDto(
     val url: String,
     val index: Int,
 )
+
+@Serializable
+data class CanReadDto(
+    val canRead: Boolean?,
+)
+
+@Serializable
+data class CommentDataDto(
+    @Serializable(with = LastStringFromArrayListSerializer::class)
+    val list: List<String>,
```

Comment:
Use `List<JsonPrimitive>` or something like that.

https://github.com/keiyoushi/extensions-source/blob/b80e212494e77a63c262056449ce67c8af70bc50/src/zh/toptoon/src/eu/kanade/tachiyomi/extension/zh/toptoon/ToptoonDto.kt#L34-L42